faboptions/src/main/java/com/joaquimley/faboptions/FabOptions.java

```diff
@@ -91,7 +95,7 @@ private void inflateButtonsFromAttrs(Context context, AttributeSet attrs) {
     }
 
     public void setButtonsMenu(Context context, @MenuRes int menuId) {
-        mMenu = new MenuBuilder(context);
+        Menu mMenu = new MenuBuilder(context); // TODO: 22/11/2016 add items in runtime
```

Comment:
The idea is to leave it as global variable so we can add items at run time, but no worries.